app/components/order-form/component.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+const { Component, inject: { service } } = Ember;
+
+export default Component.extend({
+  paperToaster: service(),
+  router: service(),
+  currentUser: service(),
+  didReceiveAttrs() {
```

Comment:
Add enter before and after to mark it as a separate part ... I usually use the given approach:

block of injected services / default properties / computed / component callbasks / actions / private methods ... all are separate via enter to make it more readable, and also each function is separate (in actions or private methods for instance, also bigger computeds).